Panic when world disconnects in background, switch to it, then switch back

Guessing index out of range error, forgot to grab log.

```
panic: runtime error: index out of range

goroutine 7 [running]:
github.com/makyo/stimmtausch/ui.(*tui).postCreate.func1(0xc420c31ad0, 0xc420b59268, 0x2)
        /build/stimmtausch-lugPab/stimmtausch-0.0.2/obj-x86_64-linux-gnu/src/github.com/makyo/stimmtausch/ui/ui.go:146 +0x294
github.com/makyo/stimmtausch/ui.(*History).Write(0xc4201c25c0, 0xc420908900, 0x2, 0x100, 0xc420055be0, 0xc420055be0, 0x40f6aa)
        /build/stimmtausch-lugPab/stimmtausch-0.0.2/obj-x86_64-linux-gnu/src/github.com/makyo/stimmtausch/ui/history.go:108 +0x154
fmt.Fprint(0x7c9020, 0xc4201c25c0, 0xc420055c40, 0x1, 0x1, 0x2, 0x3, 0x3)
        /usr/lib/go-1.10/src/fmt/print.go:223 +0x8b
github.com/makyo/stimmtausch/ui.(*tui).send(0xc4201eee70, 0xc4200fe480, 0xc4200818c0, 0xc4201ccd80, 0xc4201ccd80)
        /build/stimmtausch-lugPab/stimmtausch-0.0.2/obj-x86_64-linux-gnu/src/github.com/makyo/stimmtausch/ui/keybindings.go:29 +0xd7
github.com/makyo/stimmtausch/ui.(*tui).(github.com/makyo/stimmtausch/ui.send)-fm(0xc4200fe480, 0xc4200818c0, 0x4, 0xc420055e01)
        /build/stimmtausch-lugPab/stimmtausch-0.0.2/obj-x86_64-linux-gnu/src/github.com/makyo/stimmtausch/ui/keybindings.go:171 +0x3e
github.com/makyo/stimmtausch/vendor/github.com/makyo/gotui.(*Gui).execKeybindings(0xc4200fe480, 0xc4200818c0, 0xc420055dc0, 0x6c5744, 0x0, 0x0)
        /build/stimmtausch-lugPab/stimmtausch-0.0.2/obj-x86_64-linux-gnu/src/github.com/makyo/stimmtausch/vendor/github.com/makyo/gotui/gui.go:657 +0xce
github.com/makyo/stimmtausch/vendor/github.com/makyo/gotui.(*Gui).onKey(0xc4200fe480, 0xc420055dc0, 0xc4201c2200, 0xc420192c30)
        /build/stimmtausch-lugPab/stimmtausch-0.0.2/obj-x86_64-linux-gnu/src/github.com/makyo/stimmtausch/vendor/github.com/makyo/gotui/gui.go:621 +0x1b3
github.com/makyo/stimmtausch/vendor/github.com/makyo/gotui.(*Gui).handleEvent(0xc4200fe480, 0xc420055dc0, 0xc420055db8, 0x0)
        /build/stimmtausch-lugPab/stimmtausch-0.0.2/obj-x86_64-linux-gnu/src/github.com/makyo/stimmtausch/vendor/github.com/makyo/gotui/gui.go:438 +0x45
github.com/makyo/stimmtausch/vendor/github.com/makyo/gotui.(*Gui).MainLoopWithContext(0xc4200fe480, 0x7cc040, 0xc4200140a0, 0x0, 0x0)
        /build/stimmtausch-lugPab/stimmtausch-0.0.2/obj-x86_64-linux-gnu/src/github.com/makyo/stimmtausch/vendor/github.com/makyo/gotui/gui.go:393 +0x37c
github.com/makyo/stimmtausch/vendor/github.com/makyo/gotui.(*Gui).MainLoop(0xc4200fe480, 0x78af97, 0xd)
        /build/stimmtausch-lugPab/stimmtausch-0.0.2/obj-x86_64-linux-gnu/src/github.com/makyo/stimmtausch/vendor/github.com/makyo/gotui/gui.go:359 +0x33
github.com/makyo/stimmtausch/ui.(*tui).Run(0xc4201eee70, 0xc420066a20, 0xc420066a80)
        /build/stimmtausch-lugPab/stimmtausch-0.0.2/obj-x86_64-linux-gnu/src/github.com/makyo/stimmtausch/ui/ui.go:375 +0x3cd
created by github.com/makyo/stimmtausch/cmd.glob..func2
        /build/stimmtausch-lugPab/stimmtausch-0.0.2/obj-x86_64-linux-gnu/src/github.com/makyo/stimmtausch/cmd/root.go:91 +0x3e7
```

--- a/ui/ui.go
+++ b/ui/ui.go
@@ -144,6 +144,9 @@ func (t *tui) postCreate(g *gotui.Gui) error {
 			// the only one.
 			if line.Text[0] == '/' {
 				s := strings.SplitN(line.Text[1:], " ", 2)
+				if len(s) == 1 {
+					s = append(s, "")
+				}
 				go t.client.Env.Dispatch(s[0], s[1])
 				return nil
 			}

